pytket/extensions/azure/backends/azure.py

```diff
@@ -60,6 +60,8 @@ def _get_workspace(
     OpType.X,
     OpType.Y,
     OpType.Z,
+    OpType.ZZPhase,
```

Comment:
Unfortunately when we convert a circuit containing a ZZPhase gate to QIR using `qiskit_qir.to_qir_module()`, the gate is converted to two `CX` gates and an `Rz`. So there is no point including it here -- we may as well do the rebase ourselves. This means that circuits submitted to Quantinuum devices will _not_ be in the native gate set, and we have to rely on rebasing at the other end. This situation should improve when pytket-qir can handle the conversions ( https://github.com/CQCL/pytket-qir/issues/152 ).

Reply:
OK, will wait for this to be supported.